js/history.js

```diff
@@ -0,0 +1,126 @@
+Fliplet.Registry.set('history-stack', (function() {
```

Comment:
What does this library do? May be worth adding a comment

Reply:
Added